tools/bin/incrementversion.js

```diff
@@ -71,10 +73,10 @@ function main () {
     }
 
     // only create greater versions
-    if (!semver.gt(newVersion, oldVersion)) {
+    /*if (!semver.gt(newVersion, oldVersion)) {
```

Comment:
Please make this a simple function: something like just a check for greater major version number.